### DISCUSSION TITLE
Decimal library supports only INT32

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#368] [General]
I just noticed that the decimal library only supports INT32. CREXX supports long long, so this may be a limitation.

**Comment by adesutherland** [comment]
You are right :-(

Well - we will have to find a way around it :-) We might have to change Mike's code!

**Comment by Peter-Jacob** [comment]
Indeed, I looked at decNumberToInt32(), and it doesn't contain rocket science. My concern is that we may enter the sanctum sanctorum. We may be cursed.
